src/main/java/io/openmessaging/storage/dledger/cmdline/BossCommand.java

```diff
@@ -33,6 +34,7 @@ public static void main(String args[]) {
 
         JCommander.Builder builder = JCommander.newBuilder();
         builder.addCommand("server", new DLedgerConfig());
+        builder.addCommand("serverc", new ConfigCommand());
```

Comment:
what is the difference between serverc and server, both commands can receive a config file with -c